release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -993,21 +1092,190 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
+
+*config-operator*
+
+* Previously, the deprecated `status.platformStatus` field was not being populated during upgrade, in clusters upgraded since {product-title} 4.1. As a consequence, the upgrade could fail. This fix modified the Cluster Config Operator to populate this field. As a result, the upgrade does not fail because of this field not being populated.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1890038[*BZ#1890038*])
+
+*Console Kubevirt Plugin*
+
+* Previously, the storage class was not propagating to the VM disk list from persistent volume claims for the `DataVolume` source. The storage class is now visible in the VM disk list of the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1853352[*BZ#1853352*])
+
+* Previously, imported SR-IOV networks could be set to different network interface types. With this fix, imported SR-IOV networks are now set only to the SR-IOV network interface type. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1862918[*BZ#1862918*])
+
+* Previously, if a VM name was reused in the cluster, VM events displayed in the events screen were not correctly filtered and contained events mixed together from both VMs. Now, events are filtered properly and the events screen displays only the events belonging to the current VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1878701[*BZ#1878701*])
+
+* Previously, the `V2VVMWare` and `OvirtProvider` objects created by the VM Import wizard were not cleaned up properly. Now, the `V2VVMWare` and `OvirtProvider` objects are removed as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881347[*BZ#1881347*])
+
+* Previously, utilization data was not displayed for a Virtual Machine Interface (VMI) that did not have an associated VM. Now, if utilization data is available for a VMI, it is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884654[*BZ#1884654*])
+
+* Previously, when a PVC was cloned, its VM state was reported as *pending*, but additional information was not displayed. Now, when a PVC is cloned, the VM state is reported as *importing* along with a progress bar and additional info which contains a link to the pod or PVC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885138[*BZ#1885138*])
+
+* Previously, the VM import status displayed an incorrect VM import provider. Now, the VM import status displays the correct VM import provider. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886977[*BZ#1886977*])
+
+* Previously, the default pod network interface type was to set to the wrong value. Now, the default pod network interface type is set to masquerade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887797[*BZ#1887797*])
```

Comment:
is masquerade a value or something?

Reply:
This is apparently a term. It was used consistently in the BZ and dev PR.